panels/info/cc-info-panel.c

```diff
@@ -1798,6 +1817,72 @@ info_panel_setup_overview (CcInfoPanel  *self)
   g_clear_pointer (&os_info, g_hash_table_destroy);
 }
 
+static gboolean
+read_config_file (CcInfoPanel *self)
+{
+  GKeyFile *keyfile = NULL;
+  GError *error = NULL;
+  gchar **groups = NULL;
+  gboolean ret = FALSE;
+  gsize n_groups;
+  int i;
+
+  if (!g_file_test (CONFIG_FILEPATH, G_FILE_TEST_EXISTS))
+    return FALSE;
```

Comment:
Fold this condition when looking at the error code returned by `g_key_file_load_from_file()`